metals/src/main/scala/scala/meta/internal/builds/Digest.scala

```diff
@@ -157,7 +156,8 @@ object Digest {
   ): Boolean = {
     try {
       val input = file.toInput
-      val tokens = Trees.defaultDialect(input).tokenize.get
+      val defaultDialect = scala.meta.dialects.Scala213
```

Comment:
Scala 2.13 parser will safely work also for 2.12. There are some additional things added there, so it's rather safe. Besides, it's really just tokens, so it might not matter at all even when it comes to dotty files.

Reply:
Will change it to `Trees.defaultTokenizerDialect` to make it explicit. We could try to get the right dialect everywhere, but honestly for digest tokenizing even wrongly is fine. As long as it's stable.

Reply:
Sounds good!